data/installed-tests/fwupd.sh

```diff
@@ -13,6 +13,8 @@ run_test()
 run_device_tests()
 {
 	if [ -n "$CI_NETWORK" ] && [ -d @devicetestdir@ ]; then
+		systemctl stop fwupd.service
+		fwupdtool modify-config AllowEmulation true
```

Comment:
if you used fwupd**mgr** modify-config AllowEmulation true could you avoid stopping the service?

Reply:
I'll try.

Reply:
Got it working.